interp/interp_test.go

```diff
@@ -1038,3 +1041,34 @@ func TestRunnerOpts(t *testing.T) {
 		})
 	}
 }
+
+func TestContext(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*3)
```

Comment:
Why not make it cancel immediately? Otherwise, if I understand correctly, this test will take at least three seconds.